Neos.Neos/Resources/Private/Fusion/Root.fusion

```diff
@@ -1,2 +1,3 @@
-include: DefaultFusion.fusion
-include: RawContentMode.fusion
\ No newline at end of file
+include: RootCase.fusion
+include: RawContentMode.fusion
```

Comment:
how about we add `include: Prototypes/*.fusion` here

and remove it from RootCase :)